.idea/Max-Bot.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove the .idea folder, this is only being used to save your IDE settings and should not be added to the codebase.